reactor-core/src/test/java/reactor/core/publisher/BlockingTests.java

```diff
@@ -98,32 +99,36 @@ public void blockingLastTimeout() {
 		               .blockLast(Duration.ofMillis(1))).isNull();
 	}
 
-	@Test(expected = RuntimeException.class)
+	@Test
 	public void blockingFirstError() {
-		Flux.error(new RuntimeException("test"))
-		    .publishOn(scheduler)
-		    .blockFirst();
+		assertThrows(RuntimeException.class, () ->
```

Comment:
nit: I would avoid using short lambda syntax for multiline lambdas

Reply:
Can you clarify exactly the style you expect on a concrete case (this one for example)?
This is the result of manual editing, but always applying the same kind of repetitive keystrokes, also favoring less diff noise I thought.

Reply:
```java
public void blockingFirstError() {
	assertThrows(RuntimeException.class, () ->
			Flux.error(new RuntimeException("test"))
			    .publishOn(scheduler)
			    .blockFirst());
}
```

vs:
```java
public void blockingFirstError() {
	assertThrows(RuntimeException.class, () -> {
		Flux.error(new RuntimeException("test"))
		    .publishOn(scheduler)
		    .blockFirst();
	});
}
```
less indentation, easier to match the blocks/parenthesises.

My suggestion would also help with the diff noise, btw

Reply:
It's quite tedious sadly.

Reply:
huh? 🤔